### PR TITLE
feat: gql wasm

### PR DESCRIPTION
**Pull request by ruslan-sh-r**
## Summary by CodeRabbit

- **New Features**
	- Enhanced user contract management with new default settings.
	- Expanded query capabilities for WebAssembly (Wasm) data.

- **Improvements**
	- Updated interval settings across various default objects for consistency.

- **Documentation**
	- Public interface of the query module extended to include Wasm functionality.

### CODE REVIEW

**Review by cgilbe27 — APPROVED** [review]

File: src/indexer-nibi/query/wasm.ts [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+export interface GqlOutWasm {
+  wasm?: GQLQuery["wasm"]
+}
+
+export type GqlWasmFields = Partial<{
+  userContracts?: Partial<GQLUserContract>
+}>
+
+export const wasmQueryString = (
```
Comment (cgilbe27) [inline src/indexer-nibi/query/wasm.ts:21]: This code should likely be more like src\indexer-nibi\query\communityPool.ts

This pattern is reserved for multi-level deep modules like oracle, perps, and stats

Comment (cgilbe27) [reply src/indexer-nibi/query/wasm.ts:21]: This alternative is a more straightforward approach

**Review by cgilbe27 — CHANGES_REQUESTED** [review]
See comment above

File: src/indexer-nibi/query/wasm.ts (hunk 1 above)
Comment (ruslan-sh-r) [reply src/indexer-nibi/query/wasm.ts:21]: wasm is the same multi-level module as stats, oracle, etc.
there is only one subquery in wasm so far but can be more in the future

**Review by cgilbe27 — APPROVED** [review]

**Pull request merged**